serverless.yml

```diff
@@ -0,0 +1,39 @@
+service: webhint-staging-functions
```

Comment:
NIT: If this is the "base' config file, maybe we should remove 'staging' from the name. Same for the `resourceGroup` property

Reply:
Changed. However, the `resourceGroup` currently does not work. The changes got merged to the serverless repo, but they haven't released it. I also need to send another PR to fix some other bugs related to Azure deployment.